pytest_redis.py

```diff
@@ -1,7 +1,22 @@
 """pytest-redis queue plugin implementation."""
-
-import redis
 from _pytest.terminal import TerminalReporter
+import _pytest.runner
+import redis
+# Exit value when everything is okay
+from _pytest.main import EXIT_OK
+# Exit value when no tests are collected
+from _pytest.main import EXIT_NOTESTSCOLLECTED
+# Exit value whenever a single test fails
+from _pytest.main import EXIT_TESTSFAILED
+# Exit value when an exception is thrown by parser or KeyboardInterrupt
+from _pytest.main import EXIT_INTERRUPTED
+# Exit value when incorrect cmdline args are passed
+from _pytest.main import EXIT_USAGEERROR
+# Exit value when an internal error occurs
+from _pytest.main import EXIT_INTERNALERROR
```

Comment:
These imports could all be placed on one line (and the comments seem a bit superfluous given their names).